userguide/tutorials/reference-time-examples.adoc

```diff
@@ -0,0 +1,91 @@
+= Reference Time/Fixed Offset Timezone
+
+In this document, we look a little bit deeper into the concepts of reference time and fixed offset timezone.
+
+== Reference Time
+
+Each account in Kill Bill has a *reference time* associated with it. It can be specified at the time of creating the account. If not specified, it defaults to the account creation date time. Kill Bill uses the reference time to convert a `LocalDate` to a `DateTime` and vice-versa. So for example, if the invoice for an account is due on a particular date, the reference time is used to determine the time of the day at which the invoice will actually be generated. Note that the reference time is always considered to be in UTC.
+
+Let us take a look at a few examples of how the reference time influences invoice generation.
+
+=== Example 1
+
+* Set the time to `2023-01-01T11:00`.
+* Create an account with `referenceTime=2023-01-01T10:30`.
+* Create a subscription to a monthly plan - Invoice is generated immediately.
+* Move the clock to `2023-02-01` (Time component defaults to the current time) - Invoice is generated immediately since the current time (`11:00`) is past the reference time (`10:30`).
```

Comment:
The important part here is that this date will be interpreted in the account TZ - if specified and `UTC` otherwise.  Your example does not specify any account TZ, is this on purpose?

Reply:
Also, I am confused about the `11:00` part, how do we know current time is `11:00` - should we add this as a scenario setting, i.e `let 's assume current date/time is...`

Reply:
> The important part here is that this date will be interpreted in the account TZ - if specified and UTC 

I've now added a statement to the account creation step that states that the account TZ defaults to `UTC`. Let me know if this is still not clear.

Reply:
> Also, I am confused about the 11:00 part, how do we know current time is 11:00

The first step at the start of the example is `Set the time to 2023-01-01T11:00:00.000Z`, thus current time is `11:00`. If this is not clear, I can clarify this again in the last step.